test-packages/integration-tests/test/analytics.spec.ts

```diff
@@ -11,14 +11,9 @@ describe('usage analytics', () => {
   const hashedProjectIdentifierWithSalt =
     'e2d76b70dcb8ed5fd86e9fadf5052845ae1a0112e99ab96747b03bf78730a7ab';
 
-  const rootConfigPath = resolve(
-    __dirname
-      .split(sep)
-      .reduce((prev, curr) =>
-        prev.endsWith('integration-tests') ? prev : `${prev}${sep}${curr}`
-      ),
-    'sap-cloud-sdk-analytics.json'
-  );
+  // The cwd is different on windows and mac hence the path is set here like this
+  // The findConfigPath calls resolve() which relies on the cwd.
+  const rootConfigPath = resolve(process.cwd(), 'sap-cloud-sdk-analytics.json');
```

Comment:
> The usage analytics test fails on windows. the findConfigPath() fails the resolve() does returns the root dir on windows but the integration path on mac.
process.cwd() is different on mac and windows.

But `resolve(process.cwd(), 'sap-cloud-sdk-analytics.json');` returns the same correct path for both windows and Mac?